app/views/users/delete/show.html.erb

```diff
@@ -1,34 +1,44 @@
 <% title t('titles.confirmations.delete') %>
 
-<div class="padding-0 cntnr-xxskinny border-box bg-white rounded-xxl modal-warning">
-  <h1 class="margin-y-2 fs-20p sans-serif regular center">
-    <%= t('users.delete.heading') %>
-  </h1>
-  <hr class="margin-bottom-4 border-width-05" />
-  <div class="margin-bottom-1 bold">
-    <%= t('users.delete.subheading') %>
-  </div>
-  <ul class="padding-x-2">
-    <li class="margin-bottom-1"><%= t('users.delete.bullet_1', app_name: APP_NAME) %></li>
-    <li class="margin-bottom-1"><%= current_user.decorate.delete_account_bullet_key %></li>
-    <li class="margin-bottom-1"><%= t('users.delete.bullet_3', app_name: APP_NAME) %></li>
-    <li class="margin-bottom-1"><%= t('users.delete.bullet_4', app_name: APP_NAME) %></li>
-  </ul>
-  <%= validated_form_for(
-        current_user, url: account_delete_path,
-                      html: { autocomplete: 'off', method: :post }
-      ) do |f| %>
-    <div class="margin-bottom-4">
-      <%= t('users.delete.instructions') %>
-    </div>
-    <%= f.input :password, label: t('idv.form.password'), required: true,
-                           input_html: { aria: { invalid: false }, class: 'password-toggle' } %>
-    <%= f.button :submit,
-                 t('users.delete.actions.delete'),
-                 class: 'usa-button--big usa-button--full-width margin-bottom-2' %>
-  <% end %>
+<%= render PageHeadingComponent.new.with_content(t('users.delete.heading')) %>
 
-  <%= link_to t('users.delete.actions.cancel'), account_path,
-              role: 'button',
```

Comment:
See note concerning why we don't want `role` here:

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role#accessibility_concerns

>Warning: Be careful when marking up links with the button role. Buttons are expected to be triggered using the Space or Enter key, while links are expected to be triggered using the Enter key. In other words, when links are used to behave like buttons, adding role="button" alone is not sufficient. It will also be necessary to add a key event handler that listens for the Space key in order to be consistent with native buttons.